### PR TITLE
Make search work

### PR DESCRIPTION
**Pull request by cnunciato**
Accounts for the shift into /docs.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ generate:
 build:
 	@echo -e "\033[0;32mBUILD:\033[0m"
 	hugo
-	node ./scripts/build-search-index.js < ./public/search-data/index.json > ./public/search-index.json
+	node ./scripts/build-search-index.js < ./public/docs/search-data/index.json > ./public/docs/search-index.json
 	rm -rf ./public/search-data
```
Comment (justinvp) [inline Makefile]:
```suggestion
	rm -rf ./public/docs/search-data
```

File: layouts/docs/search-data.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+{{- $pages := (where .Site.Pages ".Params.noindex" "ne" true) -}}
```
Comment (justinvp) [inline layouts/docs/search-data.json]: Delete `layouts/_default/search-data.json` so this shows up in git as a move?

File: layouts/docs/search.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{ define "main" }}
```
Comment (justinvp) [inline layouts/docs/search.html]: Delete `layouts/_default/search.html` so this shows up in git as a move?

File: layouts/docs/search-data.json (hunk 2 above)
Comment (cnunciato) [reply layouts/docs/search-data.json]: Ah, totally thought I had. Good catch, thanks.

**Pull request merged**